auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -919,14 +798,12 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   }
 
   showNRNotFoundModal () {
-    this.$refs.addNRDialog.close()
     this.dialogTitle = 'Name Request Not Found'
     this.dialogText = 'The specified name request was not found.'
     this.$refs.errorDialog.open()
   }
 
   showNRErrorModal () {
-    this.$refs.addNRDialog.close()
```

Comment:
Had to remove these, because they're now inside of the SearchBusinessNameRequest component